packages/mdc-side-sheet/_mixins.scss

```diff
@@ -0,0 +1,216 @@
+//
+// Copyright 2017 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "@material/ripple/mixins";
+@import "@material/shape/mixins";
+@import "@material/theme/mixins";
+@import "./variables";
+
+//
+// Public
+//
+
+@mixin mdc-side-sheet-stroke-color($color, $opacity: $mdc-side-sheet-stroke-opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
+
+  @include mdc-theme-prop(border-color, $value);
+}
+
+@mixin mdc-side-sheet-shape-radius($radius, $side: trailing) {
+  @if length($radius) > 2 {
+    @error "Invalid radius: '#{$radius}'. Component doesn't allow customizing all corners.";
+  }
+
+  @if $side != "leading" and $side != "trailing" {
+    @error "mdc-side-sheet-shape-radius() mixin: $side must be 'leading' or 'trailing', but got '#{$side}'";
+  }
+
+  @if $side == "leading" {
+    @if length($radius) == 2 {
+      $radius: 0 nth($radius, 1) nth($radius, 2) 0;
+    } @else {
+      $radius: 0 $radius $radius 0;
+    }
+  } @else {
+    @if length($radius) == 2 {
+      $radius: nth($radius, 1) 0 0 nth($radius, 2);
+    } @else {
+      $radius: $radius 0 0 $radius;
+    }
+  }
+
+  @include mdc-shape-radius($radius, $rtl-reflexive: true);
+}
+
+@mixin mdc-side-sheet-scrim-fill-color($color) {
+  $value: rgba(mdc-theme-prop-value($color), $mdc-side-sheet-modal-scrim-opacity);
+
+  ~ .mdc-side-sheet-scrim {
+    @include mdc-theme-prop(background-color, $value);
+  }
+}
+
+@mixin mdc-side-sheet-ink-color($color, $opacity: $mdc-side-sheet-ink-opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
+
+  .mdc-side-sheet__title {
```

Comment:
This selector shouldn't be here, right? (This is probably why I thought the accessible fill color test page looked odd, and we don't seem to have a test page for ink color?)